articles/azure-monitor/app/opencensus-python-request.md

```diff
@@ -113,6 +113,48 @@ First, instrument your Python application with latest [OpenCensus Python SDK](./
     config = Configurator(settings=settings)
     ```
 
+## Tracking FastAPI applications
+
+1. The following dependencies are required: 
+    - fastapi
+    - uvicorn
+
+2. Add FastAPI middleware.
+
+3. Set span kind server, `span.span_kind = SpanKind.SERVER`.
+
+```python 
+app = FastAPI()
+logger = logging.getLogger(__name__)
```

Comment:
Doesn't seem like `logger` is used in the code.

Reply:
removed